pkg/greeter/greeter.go

```
@@ -0,0 +1,29 @@
+
+	"github.com/gorilla/mux"
+	"go.uber.org/fx"
+	"go.uber.org/zap"
+)
+
+// GreeterHandler provides the API for greeting
+func GreeterHandler(logger *zap.Logger) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		logger.Info("Handling Greet Request")
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte(`{"message": "Hello from Greeter!"}`))
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)